docs/faq.md

```diff
@@ -557,6 +557,22 @@ $ docker manifest inspect openjdk:8
 }
 ```
 
+You can inspect the digest (`@sha256:...`) of a manifest image with the `--verbose` option.
```

Comment:
I don't think this is necessary. Without `--verbose`, you can already see the digest of each image in the list already as seen above.